### PR TITLE
WIP: No cdts

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -16,10 +16,6 @@ if [[ "$target_platform" == osx-* ]]; then
   CONFIGURE_ARGS="${CONFIGURE_ARGS} --enable-aqua=yes"
 elif [[ "$tk_variant" == xft ]]; then
   CONFIGURE_ARGS="${CONFIGURE_ARGS} --enable-xft"
-  # Remove requires.private for now. Otherwise we need devel packages of
-  # libxrender-devel and the deps in libxrender-devel is broken anyway.
-  sed -i.bak 's/Requires.private: xrender, /Requires.private: /g' $BUILD_PREFIX/$HOST/sysroot/usr/lib/pkgconfig/xft.pc
-  pkg-config --cflags xft fontconfig
 fi
```
Comment (hmaarrfk) [inline recipe/build.sh:19]:
```suggestion
  pkg-config --cflags xft fontconfig
fi
```

**Comment by xhochy** [comment]
What is the state of this? A green CI looks promising and I have some problems with `libxau` in the python-feedstock with regards to the Alma9 migration. Thus this PR could help there.

**Comment by xhochy** [comment]
Then let's get the current main building with Alma 9 ( https://github.com/conda-forge/tk-feedstock/pull/65 ) to see whether that fixes the build issues I'm seeing in https://github.com/conda-forge/python-feedstock/pull/744

**Comment by hmaarrfk** [comment]
@isuruf would it be meaninful to consider options like: https://github.com/conda-forge/python-feedstock/issues/581 where we could better create a slimmer python for all users while avoiding CDTs?

Having this exception begs the question:

> Why should python be so "special" compared to other packages.

**Comment by hmaarrfk** [comment]
image

so 36 MB vs 16 MB, lets investigate/

**Comment by hmaarrfk** [comment]
6 MB:

    - font-ttf-dejavu-sans-mono
    - font-ttf-inconsolata
    - font-ttf-source-code-pro
    - font-ttf-ubuntu
    - fonts-conda-ecosystem
    - fonts-conda-forge

**Pull request closed without merging**